src/components/ObsDetails/ActivityItem.js

```diff
@@ -84,7 +98,10 @@ const ActivityItem = ( {
               <KebabMenu>
                 <Menu.Item
                   onPress={async ( ) => {
-                    Comment.deleteComment( item.uuid, realm, apiToken );
+                    // first delete locally
+                    Comment.deleteComment( item.uuid, realm );
+                    // then delete remotely
+                    deleteCommentMutation.mutate( item.uuid );
```

Comment:
Not a blocker, but FWIW, I think this is actually the reverse of the order we probably want for comments. Ideally, everything would be deleted remotely first, and when we're sure that worked, deleted locally, because the server is the ultimate source of truth, and also because the network request might fail, so you probably only want to alter local state when you're sure the server has been updated.

For models that have to work offline (like observations) things are more complicated b/c we have to let the app get out of sync with the server and then resolve discrepancies, but comments can be simpler.